app/views/users/show.html.erb

```diff
@@ -10,6 +10,15 @@
   </ul>
   <section>
     <h1>Bookmarked Segments</h1>
+    <% if current_user.videos %>
+    <ul>
+     <%current_user.bookmarked_videos.each do |video| %>
+      <li><%= link_to video.title, "/tutorials/#{video.tutorial_id}/?video_id=#{video.id}" %></li>
+     <% end %>
```

Comment:
It looks like this PR includes code from my bookmarked PR. we'll have to be careful